### PR TITLE
LINQPad scripts to analyze logs generated by the SDK

### PR DESCRIPTION
**Pull request by CIPop**
These are some scripts that make it easier to analyze SDK generated logs.

### CODE REVIEW

File: tools/CaptureLogs/query/readme.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+This folder contains [LINQPad](https://www.linqpad.net/) scripts that can be used in conjunction with [PerfView](https://github.com/Microsoft/perfview) XML exports to speed up SDK investigations.
+
+| Script | Description |
+| ------ | ----------- |
+|etlxml_query_summary.linq | Displays a summary of devices, modules and exceptions. |
+|etlxml_query_graph.linq | Displays logs associated with an object and it's object graph. |
+|etlxml_query_objectType.linq | Displays all logs generated by a class. |
+
+## PerfView capture details
+PerfView limits the output to 10k events. Changing that limit then exporting XML would allow all traces to be exported.
```
Comment (zolvarga) [inline tools/CaptureLogs/query/readme.md]: >exporting XML  [](start = 67, length = 14)

"exporting them to XML"

File: tools/CaptureLogs/query/readme.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## LinqPad details
+For large logs, it's recommended that the AnyCpu version of LinqPad is used.
+
+## Colorization for VSCode
+
+To read colorized logs, analyze timings use [LogFileHighlighter](https://marketplace.visualstudio.com/items?itemName=emilast.LogFileHighlighter) 
+Another recommended extension is 
+
+ Suggested color configuration :
```
Comment (zolvarga) [inline tools/CaptureLogs/query/readme.md]: >Suggested color configuration : [](start = 1, length = 31)

:thumbsup:

**Comment by CIPop** [comment]
Thanks @zolvarga , @abhipsaMisra . Made the changes based on comments. PTAL

**Pull request merged**